SlicerConfiguration/Settings/ProfileManager.cs

```diff
@@ -306,6 +306,12 @@ public PrinterSettings LoadWithoutRecovery(string profileID)
 			PrinterSettings printerSettings = Instance.LoadWithoutRecovery(profileID);
 			if (printerSettings != null)
 			{
+				// Make sure we have the name set
```

Comment:
We might also look into not black listing the printer_name field when pushing a user profile to MCWS